assets/src/util/math.js

```diff
@@ -68,11 +68,14 @@ const median = arr => {
     : sorted[middle]
 }
 
+const sum = arr => arr.reduce((acc, cur) => (acc += cur), 0)
```

Comment:
Seems like you could eliminate some duplication here, since `average` and `standardDeviation` seem to rely on a similar pattern. This isn't a high priority, though.